packages/kbn-utils/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I believe this file can be removed.

Reply:
It was automatically created when I re-built the package. You sure @tylersmalley ?

Reply:
Done :-)

Reply:
That's odd - how did you re-build the package? Looks like this one was migrated to Bazel which can be built with `yarn kbn build-bazel`

https://www.elastic.co/guide/en/kibana/master//monorepo-packages.html#_building_bazel_packages